### PR TITLE
Service type

### PR DESCRIPTION
**Pull request by ghost**
## Description
Service Type Function(Create, Edit, Delete, List)
## Reference Requirements
- Task ID: TI-27

## Reference Isuees
#27

## Migrations
YES

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

### CODE REVIEW

File: app/Http/Controllers/ServiceTypeController.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param int $id id
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/ServiceTypeController.php]: ko dùng hàm này thì xoá luôn đi

File: app/Models/ServiceType.php [hunk 2]
```diff
@@ -29,4 +31,73 @@ public function services()
...
+    /**
+     * Find Service Type from id
+     *
+     * @param int $id id
+     *
+     * @return array
+    */
+    public function findServiceType($id)
+    {
+        return $this->where('id', $id)->first();
```
Comment (at-tuannguyen3) [inline app/Models/ServiceType.php]: thay vì dùng where trỏ tới firt thì dùng find luôn cho gọn $this->find($id)

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment! tks you

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**